clipboard-extension.md

```diff
@@ -0,0 +1,44 @@
+# Clipboard Extension
+
+The Clipboard Extension supports copying to the clipboard using a subset of the XTerm OSC 52 escape sequence. 
+
+## Feature detection
+
+Support for this functionality in a terminal is indicated by the extension parameter 52 in the Device Attributes (DA) report.
+
+## Syntax
+The escape sequence has the following syntax:
+
+```
+OSC 52 ; Pc ; Pd ST
+```
+
+Conforming terminals MUST at least support an empty Pc value, and a Pc value of c. Other values are optional.
+
+Conforming terminals MUST support a Pd value that is a base64 encoded representation of the content that will be copied to the clipboard. They SHOULD also support an empty Pd value to clear the clipboard. They MAY support a Pd value of ? to query the clipboard content (as specified in the XTerm documentation).
+
+If a terminal has an option to disable clipboard access, it MUST only indicate support for this extension when writing to the clipboard is actually allowed.
+
+### Adoption State
+
+| Support | Terminal/Tookit/App                                        | Notes |
+|---------|------------------------------------------------------------|-------|
+| not yet | [Alacritty](https://github.com/alacritty/alacritty)        | |
+| not yet | [Contour](https://github.com/contour-terminal/contour/)    | [contour#1765](https://github.com/contour-terminal/contour/issues/1765) |
+| not yet | [DomTerm](https://github.com/PerBothner/DomTerm)           | [doomterm#124](https://github.com/PerBothner/DomTerm/issues/124) |
+| ✅      | [foot](https://codeberg.org/dnkl/foot)                     | [foot#2130](https://codeberg.org/dnkl/foot/pulls/2130) |
+| not yet | [ghostty](https://github.com/ghostty-org/ghostty)          | [ghostty#7590](https://github.com/ghostty-org/ghostty/discussions/7590) 
+| not yet | [iTerm2](https://github.com/gnachman/iTerm2)               | |
+| not yet | [Kitty](https://github.com/kovidgoyal/kitty)               | |
+| not yet | [Konsole](https://konsole.kde.org/)                        | |
+| not yet | [mintty](https://github.com/mintty/mintty)                 | |
+| not yet | [mlterm](https://github.com/arakiken/mlterm)               | [mlterm#144](https://github.com/arakiken/mlterm/issues/144) |
+| not yet | [notcurses](https://github.com/dankamongmen/notcurses)     | |
+| not yet | [st](https://st.suckless.org/)                             | |
+| ✅      | [Terminal](https://github.com/ismail-yilmaz/Terminal)      | [commit](https://github.com/ismail-yilmaz/Terminal/commit/4f62f0fd688d91e802779c30342bc5faceda50ef)|
+| not yet | [VTE](https://gitlab.gnome.org/GNOME/vte) / gnome-terminal | |
+| not yet | [Wezterm](https://github.com/wez/wezterm)                  | |
+| not yet  | [Windows Terminal](https://github.com/microsoft/terminal/)| [wt#2946](https://github.com/microsoft/terminal/issues/2946) Indicate support for OSC 52 in the DA1 report  by @j4james : [wt#19034](https://github.com/microsoft/terminal/pull/19034) |
```

Comment:
For the notes here, I'd just link to the PR for the extension, i.e. `wt#19034`. I don't there's any need to include the issue for the original `OSC 52` implementation.